config/profile/rhdh/default-config/service.yaml

```diff
@@ -9,4 +9,8 @@ spec:
   ports:
     - name: http-backend
       port: 80
-      targetPort: backend
\ No newline at end of file
+      targetPort: backend
+    - name: http-metrics
```

Comment:
Is this port exposed only in our RHDH/Showcase image? Or is this also exposed out of the box from a raw Backstage image?
(Trying to see if we also need to update the Service for the [`backstage.io` profile](https://github.com/redhat-developer/rhdh-operator/tree/main/config/profile/backstage.io)).

Reply:
Yes, the port is also exposed upstream [here](https://github.com/backstage/backstage/blob/7d8777dbba10cabe0629e8f239806231519f5062/packages/backend/src/instrumentation.js#L26). Will update my PR to include these changes for the backstage.io profile as well 👍 

Thank you for the review!